contracts/src/v0.8/dev/functions/Functions.sol

```diff
@@ -63,6 +64,9 @@ library Functions {
     }
 
     if (self.secrets.length > 0) {
+      if (self.secretsLocation == Location.Inline) {
```

Comment:
This is not enough to prevent calling oracle.sendRequest() with inline secrets. Where will be the other check? If not in the Oracle contract then either in the node or in the adapter (easiest?).

Reply:
The current plan is for it to be in the EA, which means that the cutover to the breaking version will be when NOP re-deploy their EAs

Reply:
Got it, so is the plan to deploy contracts after that happens?

Reply:
Regarding inline secrets there are no contract deployments that CLL needs to handle. The only thing that changed is within a user's consuming contract.

But yeah, after inline secrets is resolved I'll do a redeployment just to get everything up to date with the contracts in this repo.